content/hardware/02.hero/boards/uno-r4-wifi/tutorials/opamp/opamp.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Arduino UNO R4 WiFi OPAMP'
+description: 'Learn how to use the built-in Operational Amplifier in the UNO R4 WiFi'
+tags:
+  - OPAMP
+author: 'Maximilian Gerhardt, Hannes Siebeneicher'
+hardware:
+  - hardware/02.hero/boards/uno-r4-wifi
+software:
+  - web-editor
+  - ide-v1
+  - ide-v2
+---
+
+In this tutorial, you will learn how to use the built-in operational amplifier (OPAMP) featured on the [Arduino UNO R4 WiFi](https://store.arduino.cc/products/uno-r4-wifi). Operational amplifiers are very versatile and can be used to e.g. mirror an electrical signal or amplify it. 
+
+## Goals
+
+In this article, you will learn:
+- about the OPAMP feature onboard the UNO R4 WiFi,
+- about the basics of the OPAMP library, 
+- how to mirror a voltage signal,
+- how to amplify a voltage signal.
+
+## Hardware & Software Needed
+  To follow along with this article, you will need the following hardware: 
+  
+  - [Arduino UNO R4 WiFi](https://store.arduino.cc/uno-r4-wifi)
+  - resistors 
```

Comment:
What value?

Reply:
Depends on the amplification. I think the section explaining how to calculate the resistor value should be helpful.